packages/apps/tools/src/components/Global/AddNetworkModal/AddNetworkModal.tsx

```diff
@@ -15,9 +15,9 @@ import { useForm } from 'react-hook-form';
 import * as z from 'zod';
 
 const schema = z.object({
-  label: z.string().trim(),
-  networkId: z.string().trim(),
-  api: z.string().trim(),
+  label: z.string().trim().min(1),
```

Comment:
Do we have any more specific requirements for the input (instead of just minimal 1 character)?

Reply:
Not than I know of, let me know if you think some more should be added